modules/dumbCommandModule/index.js

```diff
@@ -1,3 +1,5 @@
+var _ = require('underscore');
```

Comment:
I prefer lodash and I've started using the character `u` instead of `_` because using `_` will cause broken behavior if you copy-paste such code into the nodejs repl (where `_` means the last expression's result).

No big deal though, and I know I've got a lot of `_` floating around still, just felt like throwing that out there. Still probably merging this.